### ISSUE TITLE
Load huge data in keras for training.

### ISSUE THREAD

**Issue by sraj88** [keras-team/keras#9145]
I am a beginner in Keras. I wanted to load data (dim of 492x1100k approx. delimited by commas(,)) for training but the data is around 4.2GB. I am not able to load in keras. The machine gets stuck all the time. I am using GPU of size 8GB. Any suggestions would be really appreciable..

**Comment by lorenzori** [comment]
https://github.com/keras-team/keras/issues/7729

**Comment by rex-yue-wu** [comment]
Alternatively, you can use [numpy's memory mapping mode](https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.memmap.html). To do so, you need to 
1. convert your data as a numpy array
2. load this array from disk using memory mapping
